index.html

```diff
@@ -83,62 +89,83 @@ <h2>Main Program</h2>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 3 -->
+            <div class="card-content dflex">
+              <img src="./images/chat-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Forum</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 4 -->
+            <div class="card-content dflex">
+              <img src="./images/main-event-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Workshop</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 5 -->
 
-          <div class="card-content dflex">
-            <img src="./images/ignite-icon.png" alt="conference-icon" />
-            <h3 class="prg-title prg-1">Network</h3>
-            <p>
-              Listen to the speakers from various countries about the messages
-              of sharing and opening.
-            </p>
-          </div>
-          <div class="desktop-only">
-            <a href="#">SEE THE WHOLE PROGRAM</a>
+            <div class="card-content dflex">
+              <img src="./images/ignite-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Network</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
           </div>
         </div>
-        <div class="join">Join global DevCon 2022</div>
+        <div class="desktop-only see-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="join mobile-only">Join global DevCon 2022</div>
       </div>
       <div class="speakers-section"></div>
     </main>
+    <section id="sponsors" class="sponsors desktop-only dflex">
+      <h2 class="centered-text">Partners</h2>
+      <div class="line line2"></div>
+      <div class="sponsors-logos dflex">
+        <img src="./images/mozilla.png" alt="mozilla" />
+        <img src="./images/Google.png" alt="Google" />
+        <img src="./images/navar.png" alt="navar" />
+        <img src="./images/daumkakao.svg" alt="daumkakao" />
+        <img src="./images/airbnb.png" alt="airbnb" />
+      </div>
+    </section>
+    <!-- Footer -->
+    <footer class="footer dflex desktop-only">
+      <div class="footer-logo">
+        <img src="./images/devlogo.png" alt="logo" />
+      </div>
+      <h3>&#169; 2022 DevCon, some rights reserved</h3>
+      <div class="pattern"></div>
+    </footer>
     <!-- Footer -->
-    <footer class="footer dflex">
+    <footer class="footer mobile-only dflex">
```

Comment:
For better semantics, kindly ensure that the body only contains a single footer element.